### PR TITLE
Add total duration for unplayed items.

### PR DESCRIPTION
**Pull request by mstratford**

### CODE REVIEW

File: src/showplanner/Player.tsx [hunk 1]
```diff
@@ -222,11 +222,15 @@ export function Player({ id }: { id: number }) {
     }
   };
 
-  var duration: number = 0;
+  var channel_duration: number = 0;
+  var channel_unplayed: number = 0;
```
Comment (markspolakovs) [inline src/showplanner/Player.tsx:226]: * camelCase
* no need for explicit typedef

File: src/showplanner/Player.tsx [hunk 2]
```diff
@@ -239,6 +243,10 @@ export function Player({ id }: { id: number }) {
       }
     >
       <div className="card text-center">
+        <span>
+          Total: {secToHHMM(channel_duration)}&nbsp;<strong>-</strong>
+          &nbsp;Un-played: {secToHHMM(channel_unplayed)}
```
Comment (markspolakovs) [inline src/showplanner/Player.tsx:248]: but y tho

**Review by markspolakovs — CHANGES_REQUESTED** [review]

File: src/showplanner/Player.tsx (hunk 2 above)
Comment (mstratford) [reply src/showplanner/Player.tsx]: It was ignoring spaces. Corrected now.

**Review by markspolakovs — APPROVED** [review]
I mean the "but y tho" was more about the entire concept of the PR - but sure, feature-creep away :p

**Comment by mstratford** [comment]
> I mean the "but y tho" was more about the entire concept of the PR - but sure, feature-creep away :p

Idea was that if you needed to know how much music you have left to play during your show :)

**Pull request merged**